src/generated/resources/assets/enderio/lang/zh_cn.json

```
@@ -0,0 +1,1108 @@
+  "block.enderio.clear_glass_nm_red": "红色纯净玻璃",
+  "block.enderio.clear_glass_nm_white": "白色纯净玻璃",
+  "block.enderio.clear_glass_nm_yellow": "黄色纯净玻璃",
+  "block.enderio.clear_glass_np": "纯净玻璃",
+  "block.enderio.clear_glass_np_black": "黑色纯净玻璃",
+  "block.enderio.clear_glass_np_blue": "蓝色纯净玻璃",
+  "block.enderio.clear_glass_np_brown": "棕色纯净玻璃",
+  "block.enderio.clear_glass_np_cyan": "青色纯净玻璃",
+  "block.enderio.clear_glass_np_gray": "灰色纯净玻璃",
+  "block.enderio.clear_glass_np_green": "绿色纯净玻璃",
+  "block.enderio.clear_glass_np_light_blue": "淡蓝色纯净玻璃",
+  "block.enderio.clear_glass_np_light_gray": "淡灰色纯净玻璃",
+  "block.enderio.clear_glass_np_lime": "黄绿色纯净玻璃",
+  "block.enderio.clear_glass_np_magenta": "品红色纯净玻璃",
+  "block.enderio.clear_glass_np_orange": "橙色纯净玻璃",
+  "block.enderio.clear_glass_np_pink": "粉红色纯净玻璃",
+  "block.enderio.clear_glass_np_purple": "紫色纯净玻璃",
+  "block.enderio.clear_glass_np_red": "红色纯净玻璃",
+  "block.enderio.clear_glass_np_white": "白色纯净玻璃",
+  "block.enderio.clear_glass_np_yellow": "黄色纯净玻璃",
+  "block.enderio.clear_glass_orange": "橙色纯净玻璃",
+  "block.enderio.clear_glass_p": "纯净玻璃",
+  "block.enderio.clear_glass_p_black": "黑色纯净玻璃",
+  "block.enderio.clear_glass_p_blue": "蓝色纯净玻璃",
+  "block.enderio.clear_glass_p_brown": "棕色纯净玻璃",
+  "block.enderio.clear_glass_p_cyan": "青色纯净玻璃",
+  "block.enderio.clear_glass_p_gray": "灰色纯净玻璃",
+  "block.enderio.clear_glass_p_green": "绿色纯净玻璃",
+  "block.enderio.clear_glass_p_light_blue": "淡蓝色纯净玻璃",
+  "block.enderio.clear_glass_p_light_gray": "淡灰色纯净玻璃",
+  "block.enderio.clear_glass_p_lime": "黄绿色纯净玻璃",
+  "block.enderio.clear_glass_p_magenta": "品红色纯净玻璃",
+  "block.enderio.clear_glass_p_orange": "橙色纯净玻璃",
+  "block.enderio.clear_glass_p_pink": "粉红色纯净玻璃",
+  "block.enderio.clear_glass_p_purple": "紫色纯净玻璃",
+  "block.enderio.clear_glass_p_red": "红色纯净玻璃",
+  "block.enderio.clear_glass_p_white": "白色纯净玻璃",
+  "block.enderio.clear_glass_p_yellow": "黄色纯净玻璃",
+  "block.enderio.clear_glass_pink": "粉红色纯净玻璃",
+  "block.enderio.clear_glass_purple": "紫色纯净玻璃",
+  "block.enderio.clear_glass_red": "红色纯净玻璃",
+  "block.enderio.clear_glass_white": "白色纯净玻璃",
+  "block.enderio.clear_glass_yellow": "黄色纯净玻璃",
+  "block.enderio.cloud_seed": "云之精华",
+  "block.enderio.cloud_seed_concentrated": "浓缩云之精华",
+  "block.enderio.cold_fire": "寒焰",
+  "block.enderio.conductive_alloy_block": "导电合金块",
+  "block.enderio.conduit": "导管",
+  "block.enderio.copper_alloy_block": "铜合金块",
+  "block.enderio.crafter": "合成器",
+  "block.enderio.creative_power": "创造能源",
+  "block.enderio.dark_steel_bars": "玄钢栏杆",
+  "block.enderio.dark_steel_block": "玄钢块",
+  "block.enderio.dark_steel_door": "玄钢门",
+  "block.enderio.dark_steel_ladder": "玄钢梯子",
+  "block.enderio.dark_steel_pressure_plate": "玄钢压力板",
+  "block.enderio.dark_steel_trapdoor": "玄钢活板门",
+  "block.enderio.dew_of_the_void": "深渊露水",
+  "block.enderio.enchanter": "附魔器",
+  "block.enderio.end_steel_bars": "末地钢栏杆",
+  "block.enderio.end_steel_block": "末地钢块",
```

```suggestion
  "block.enderio.end_steel_block": "末地钢块",
  "block.enderio.enderman_head": "末影人的头",
```
原文：**Enderman Head**